src/orders/service/JsonOrderFormatterService.ts

```diff
@@ -26,8 +28,11 @@ export default class JsonOrderFormatterService {
     }
     order.totalTax = Number(order.totalTax);
     order.totalGrams = Number(order.totalGrams);
+    order.customer = trimObject(order.customer) as Customer;
```

Comment:
@mb-ca @mvx24 , Sorry, im a little out of context here, but wanted to ask whiles i was here,  is there a reason we'd only do this on the customer object, and not the rest of the objects on the order for the sake of consistency? Seems a little weird that customer fields would get trimmed but the others wouldn't.

Reply:
@ryankazokas @mb-ca  It seems like that's the only one with end-user input involved.  If we can't trust other non-user entered data from third-party systems then we should be doing this everywhere not just on Orders.

Reply:
@mvx24 sorry, not really sure about the use case, but this particular function takes in any order object from anywhere. Not necessarily user-inputted data. All of the data on the order itself can be written elsewhere and passed in here(like we are in playbook). I guess all I'm concerned about here though is the implicit behavior of trimming the strings on order.customer but not order.line _item or order.fulfillment. Just didn't know if it fit in the SDK or should be in the integration itself that is dealing with the order since changing it here will have a ripple affect across all integrations. 

Again, i don't see a use case where having the extra spaces is necessary(except maybe doing strict diffs), so I'm not saying i disagree with the change, but am moreso advocating for the consistency of the order object that is output from the function.

Reply:
@ryankazokas I was told there was specific errors originating from bad user input.  I've changed the code to process the whole object now.